src/documents/en/faq.html.md

```diff
@@ -30,5 +30,10 @@ We also perform daily backups of your data, so we can restore them if needed.
 
 We provide 40GB of storage with each instance (well, technically 60GB are available, but we recommend for now not to use more than 40GB)
 
+## What can i do if I forget/loose my password ?
+
+For security and confidentiality reasons, we never have access to your decrypted password, and as a result, the datas of your different apps will be lost. But happely enough, not your cozy ! The only solution is to create a new password by retyping it in the "password recovery email", that you can activate in the login page of your cozy ("Forget password? Email me instructions"). You will then have to reinitialize all your apps.
```

Comment:
This isn't very clear and partially false. Only the encrypted data will be lost, and we don't encrypt everything, only passwords. So when user forget its password, he/she can request a new one, but this means he/she will have to re-enter the passwords in every application that use passwords, for example Emails, Konnectors, Kresus…